src/Cake/Cake.csproj

```diff
@@ -0,0 +1,82 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is there a reason that this csproj doesn't have the Authors element like the others?